lib/core/regs/consent.test.js

```diff
@@ -15,6 +15,53 @@ const denyAll = {
   measureAdvertisingPerformance: false,
 };
 
+describe("applicableReg", () => {
+  it("returns gdpr when cmp indicates gdpr applies", () => {
+    expect(applicableReg("can", { gdprApplies: true })).toBe("gdpr");
+    expect(applicableReg("us", { gdprApplies: true })).toBe("gdpr");
+    expect(applicableReg(null, { gdprApplies: true })).toBe("gdpr");
+
+    expect(applicableReg("can", { gppSectionIDs: [2] })).toBe("gdpr");
+    expect(applicableReg("us", { gppSectionIDs: [2] })).toBe("gdpr");
+    expect(applicableReg(null, { gppSectionIDs: [2] })).toBe("gdpr");
+  });
+
+  it("returns can when cmp indicates can applies", () => {
+    expect(applicableReg("gdpr", { gppSectionIDs: [5] })).toBe("can");
+    expect(applicableReg("us", { gppSectionIDs: [5] })).toBe("can");
+    expect(applicableReg(null, { gppSectionIDs: [5] })).toBe("can");
+  });
+
+  it("returns us when cmp indicates us applies", () => {
+    expect(applicableReg("gdpr", { gppSectionIDs: [7] })).toBe("us");
+    expect(applicableReg("can", { gppSectionIDs: [7] })).toBe("us");
+    expect(applicableReg(null, { gppSectionIDs: [7] })).toBe("us");
```

Comment:
Can you add test cases for multiple US section IDs (which is the case when there should be multiple passed)

Reply:
Updated